src/Views/CategoryView.vala

```diff
@@ -45,17 +46,21 @@ public class AppCenter.CategoryView : Gtk.Stack {
         };
 
         scrolled = new Gtk.ScrolledWindow (null, null) {
+            child = box,
             hscrollbar_policy = Gtk.PolicyType.NEVER
         };
-        scrolled.add (box);
 
         var spinner = new Gtk.Spinner () {
-            halign = Gtk.Align.CENTER
+            halign = Gtk.Align.CENTER,
+            hexpand = true
         };
         spinner.start ();
 
-        add (spinner);
-        add (scrolled);
+        stack = new Gtk.Stack ();
+        stack.add (spinner);
+        stack.add (scrolled);
+
+        add (stack);
```

Comment:
Should this not be `child = stack` ?

Reply:
I don't think we can use `child` since a box can have more than one child?